pkgs/development/tools/misc/texinfo/common.nix

```diff
@@ -49,7 +49,7 @@ stdenv.mkDerivation {
   # A native compiler is needed to build tools needed at build time
   depsBuildBuild = [ buildPackages.stdenv.cc perl ];
 
-  buildInputs = [ xz.bin bash libintl ]
+  buildInputs = [ (lib.getBin xz) bash libintl ]
```

Comment:
This seems wrong to me especially for cross compilation

Reply:
I think @SuperSandro2000 means this should be a `nativeBuildInput`

Reply:
> Is your comment about `xz.bin` in the first place

yes

Reply:
We don't need that here anymore. At the time the tarball was in the xz format but now we have gz.

Reply:
We don't need that here anymore. At the time the tarball was in the xz format but now we have gz and strictDeps is already set.